### PR TITLE
Update doc for adding new param in cat shards action for cancellation…

### PR DESCRIPTION
**Pull request by aasom143**
… support

### Description
Cancellation support on cat shards admin API. Added new flag to set the timeout for shard request.

### Issues Resolved
Closes #13908

### Version
From OS version 2.17 and onwards, the cancellation support will be given to cat shards API. 

### Checklist
- [ ] By submitting this pull request, I confirm that my contribution is made under the terms of the Apache 2.0 license and subject to the [Developers Certificate of Origin](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).
For more information on following Developer Certificate of Origin and signing off your commits, please check [here](https://github.com/opensearch-project/OpenSearch/blob/main/CONTRIBUTING.md#developer-certificate-of-origin).

### CODE REVIEW

File: _api-reference/cat/cat-shards.md [hunk 1]
```diff
@@ -33,6 +33,7 @@ Parameter | Type | Description
 bytes | Byte size | Specify the units for byte size. For example, `7kb` or `6gb`. For more information, see [Supported units]({{site.url}}{{site.baseurl}}/opensearch/units/).
 local | Boolean | Whether to return information from the local node only instead of from the cluster manager node. Default is `false`.
 cluster_manager_timeout | Time | The amount of time to wait for a connection to the cluster manager node. Default is 30 seconds.
+cancel_after_time_interval | Time | The time after which the shards request will be canceled. Default is -1.
```
Comment (kolchfa-aws) [inline _api-reference/cat/cat-shards.md:36]:
```suggestion
cancel_after_time_interval | Time | The amount of time after which the shard request will be canceled. Default is `-1`.
```

**Review by kolchfa-aws — APPROVED** [review]
Thank you, @aasom143! Just one suggestion.

File: _api-reference/cat/cat-shards.md (hunk 1 above)
Comment (aasom143) [reply _api-reference/cat/cat-shards.md:36]: I was referring the flag from the cat search API, Can you please check that - https://opensearch.org/docs/latest/api-reference/search/
If you still want i will make above suggestion.

Comment (kolchfa-aws) [reply _api-reference/cat/cat-shards.md:36]: This way it's clearer that it's the amount of time and not just a time on the clock. We should change in search API too.

**Comment by aasom143** [comment]
@kolchfa-aws can you please check and merge if it looks good?

**Review by kolchfa-aws — APPROVED** [review]
LGTM

**Comment by kolchfa-aws** [comment]
Thank you, @aasom143! Applying "Done but waiting to merge" label to put this in the 2.17 editorial queue starting 9/10. This PR will be merged before the release.

**Review by natebower — APPROVED** [review]
LGTM

**Pull request merged**